R/REST_helpers.R

```diff
@@ -91,14 +95,24 @@ sbtools_DELETE <- function(url, ..., session) {
 # HEAD fxn
 sbtools_HEAD <- function(url, ..., session) {
 	session_val(session)
-	r <- HEAD(url = url, ..., httrUserAgent(), handle = session)
+	r <- tryCatch(HEAD(url = url, ..., httrUserAgent(), handle = session),
+					 error = function(e) {
+							warning(paste("Something went wrong with request: \n",
```

Comment:
So it looks like this will only give a warning instead of stop if the error isn't 'Item not found'.  Is that correct, and the intended behavior?

Reply:
More or less -- The only error we "expect" is item not found. Any other network error should, according to CRAN policy, not throw and error.

Reply:
but you get `item not found` when you aren't authorized to access an item. Is that really what you want to convert into a warning too?

Reply:
Actually -- I think we are looking at two different things. In the "HEAD" case, any error from the HEAD request will get raised as a warning and r gets list(status_code = 400) which then causes the function to return FALSE.

Up above, in `sbtools_GET` any error other than "item not found..." is raised as a warning.